src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      calculator.result += value;
+
+      return this;
+    },
+    subtract(value) {
```

Comment:
its better to put an empty line between methods for better readability

Reply:
Agree